exporter/awsemfexporter/config.go

```diff
@@ -41,13 +41,16 @@ type Config struct {
 	ProxyAddress string `mapstructure:"proxy_address"`
 	// Region is the AWS region where the metric logs are sent to.
 	Region string `mapstructure:"region"`
-	// Amazon Resource Name (ARN) of the AWS resource running the collector.
-	ResourceARN string `mapstructure:"resource_arn"`
 	// RoleARN is the IAM role used by the collector when communicating
 	// with the CloudWatch Logs service
 	RoleARN string `mapstructure:"role_arn"`
 	// NoVerifySSL is the option to disable TLS certificate verification.
 	NoVerifySSL bool `mapstructure:"no_verify_ssl"`
 	// MaxRetries is the maximum number of retries before abandoning an attempt to post data.
 	MaxRetries int `mapstructure:"max_retries"`
+	// DimensionRollupOption is the option for metrics dimension rollup. Three options are available, default option is 0.
+	// 0 - Enable both zero dimension rollup and single dimension rollup
+	// 1 - Enable single dimension rollup
+	// 2 - No dimension rollup (only keep original metrics which contain all dimensions)
+	DimensionRollupOption int `mapstructure:"dimension_rollup_option"`
```

Comment:
We plan to introduce the meaning of these int in the User Guide. We were thinking it could introduce some typos if we let customers directly use string here.

Reply:
I think even an int can have typos, wrong int ;) But configs like code are read at least as much as written so I think readability of the config file is a good idea.

Reply:
Makes sense. I will change it to string to be more readable.